cmd/ocitools/validate.go

```diff
@@ -84,7 +84,7 @@ var bundleValidateCommand = cli.Command{
 			return fmt.Errorf("The root path %q is not a directory.", rootfsPath)
 		}
 
-		hostCheck := context.Bool("host-specific")
+		hostCheck := hostSpecific || context.Bool("host-specific")
```

Comment:
I don't think we want both a global and a per-command --host-specific here.  We should drop the per-command option.

And I'm not all that familiar with urfave/cli, but do we really need the package-global `hostSpecific`?  I'd expect you to be able to pull that from the `context`, even though it's set via a global variable.